epsagon_kubernetes_exporters_deploy.sh

```diff
@@ -0,0 +1,93 @@
+#! /bin/bash
+function is_positive_answer {
+    answer=$1
+    if [ ${answer} == 'y' ] ; then
+        answer='Y'
+    fi
+    if [ ${answer} == 'Y' ] ; then
+        return 0;
+    fi
+    return 1;
+}
+
+function install_exporter {
+    CONTEXT=$1
+    CONFIG=$2
+    EXPORTER_FILENAME=$3
+    KUBECTL="kubectl --context ${CONTEXT} --kubeconfig=${CONFIG}"
+    ${KUBECTL} apply -f ${EXPORTER_FILENAME}
+    rm -f ${EXPORTER_FILENAME}
+}
+
+function apply_celery_exporter {
+    CONTEXT=$1
+    CONFIG=$2
+    echo 'Please insert the messaging broker url used by your Celery, for example:'
+    echo "rabbitmq: amqp://guest:guest@rabbitmq-service.monitoring:5672//"
+    echo "redis: redis://redis:6379/"
+    echo "URL:"
+    read broker_url
+    cat external-metrics-exporters/celery-exporter.yaml | sed -e "s|\${CELERY_BROKER_URL}|${broker_url}|g" > metrics_exporter.yaml
+    install_exporter $CONTEXT $CONFIG "metrics_exporter.yaml"
+}
+
+function apply_node_exporter {
+    CONTEXT=$1
+    CONFIG=$2
+    cp external-metrics-exporters/node-exporter.yaml ./metrics_exporter.yaml
```

Comment:
I'd use a different name for the file here and in the celery installation (instead of `./metrics_exporter.yaml` in both) just to make sure we avoid overwriting or some other weird bugs if an action fails